common/djangoapps/tedix_ro/views.py

```diff
@@ -197,8 +199,16 @@ def manage_courses(request):
                         send_mail(subject, txt_message, from_address, [parent.user.email], html_message=html_message)
 
                 if form.cleaned_data['send_sms']:
-                    # sending sms logic to be here
-                    pass
+                    parent = student.parents.first()
+                    context.update({
+                        'student_name': student.user.profile.name or student.user.username,
+                        'courses_due_dates_url': urljoin(settings.LMS_ROOT_URL, reverse('personal_due_dates')),
```

Comment:
using `settings.LMS_ROOT_URL` will break sites (if any will be configured)

preferred way is to use `configuration_helpers`
```
from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
```

Reply:
This project isn't intended to be configured for microsites.